### PR TITLE
Fix #669

### PR DESCRIPTION
**Pull request by lazypanda10117**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://lazypanda10117.github.io/gci15.fossasia.org

File: _data/gallery.yml [hunk 1]
```diff
@@ -227,7 +227,13 @@
...
+  
   -
     name: File Metadata
     image: filemetadataanshuman73.png
-    author: Anshuman Agarwal 
\ No newline at end of file
+    author: Anshuman Agarwal 
+
+  -
+    name: PHP App: Hello Name
```
Comment (jigyasa-grover) [inline _data/gallery.yml]: I am guessing the build is failing owing to `:` in `name: PHP App: Hello Name`

The **build failure** message reads as: `"jekyll 2.4.0 | Error:  (/home/travis/build/fossasia/gci15.fossasia.org/_data/gallery.yml): mapping values are not allowed in this context at line 237 column 18"`

**Comment by jigyasa-grover** [comment]
Also squash your commits...

File: index.html [hunk 2]
```diff
@@ -1376,3 +1381,5 @@ <h2 class="section-h1"><span class="white-border">Social Media</span></h2>
 </script>
 </body>
 </html>
+Status API Training Shop Blog About Pricing
```
Comment (samarsault) [inline index.html:1384]: What are these 2 lines for ?

Comment (jigyasa-grover) [inline index.html:1384]: Seems to be taken from the footer of [github.com](www.github.com) :P

Comment (mananwason) [inline index.html:1384]: Yes @jig08 It seems to copied from some github page only :stuck_out_tongue:

**Comment by jigyasa-grover** [comment]
@lazypanda10117 Fixing this ?

**Comment by rkgudboy** [comment]
@lazypanda10117  Please resolve the merge conflicts.

**Comment by abishekvashok** [comment]
@lazypanda100117 lease resoplve the conflicts see here https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md for more info about how to resolve conflicts

**Pull request closed without merging**